dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -739,148 +768,213 @@ public String getWorkflowDag(@ApiParam(value = "workflowId", required = true) @P
                 e.printStackTrace();
```

Comment:
Same here, read content from memory rather than writing it to disk and back again to avoid possibility of IOExceptions.